config/src/config.rs

```diff
@@ -1464,3 +1467,30 @@ impl Default for ExitBehavior {
         ExitBehavior::CloseOnCleanExit
     }
 }
+
+#[derive(Debug, Deserialize, Serialize, Clone, Copy, PartialEq, Eq)]
+pub enum DroppedFileQuoting {
+    /// Do no quoting at all, just pass through the name as-is
```

Comment:
```suggestion
    /// Do not do any quoting at all, just pass through the name as-is
```

Reply:
The former is actually better grammar, but perhaps the issue is just that English is weird :-p
Perhaps it is better to rephrase as "No quoting is performed, the file name is passed through as-is".